File: Listas de Exercícios (Back-end [SPOLOGP])/Python/Lista de Reforço/8.py
print("Tarifa\n")

S=float(input("Saldo: "))

if (S>=1000 and S<3000):
    S=S*1.8
elif (S>=3000 and S<5000):
    S=S*1.55
elif (S>=5000):
    S=S*1.25

print(f"\nSaldo final: {S}\n")

File: Listas de Exercícios (Back-end [SPOLOGP])/C/Lista de Reforço/8.c
#include <stdio.h>

int main() {
    float S;

    printf("Tarifa\n\n");

    printf("Saldo: "); scanf("%f", &S);

    if (S>=1000 && S<3000){
        S=S*1.8;
    }else if (S>=3000 && S<5000){
        S=S*1.75;
    }else if (S>=5000){
        S=S*1.7;
    }

    printf("\nSaldo final %.2f\n", S);

    return 0;
}